rapids-4-spark-tools/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ClassWarehouse.scala

```diff
@@ -44,4 +44,65 @@ case class ExecutorCase(
   executorID: String, host: String, totalCores: Int, resourceProfileId: Int)
 
 case class ExecutorRemovedCase(
-  executorID: String, reason: String, time: Long)
\ No newline at end of file
+  executorID: String, reason: String, time: Long)
+
+case class SQLExecutionCase(
+    sqlID: Long, description: String, details: String,
+    startTime: Long, endTime: Option[Long],
+    duration: Option[Long], durationStr: String)
+
+case class SQLPlanMetricsCase(
+    sqlID: Long, name: String, accumulatorId: Long,
+    metricType: String)
+
+case class PlanNodeAccumCase(
+    sqlID: Long, nodeID: Long, nodeName:String,
+    nodeDesc: String, accumulatorId: Long)
+
+case class DriverAccumCase(
+    sqlID: Long, accumulatorId: Long, value: Long)
+
+case class TaskStageAccumCase(
+    stageId: Int, attemptId: Int, taskId: Option[Long],
```

Comment:
If we can't fit the params on 2 lines then we should have one param per line, based on https://github.com/databricks/scala-style-guide#spacing-and-indentation

Reply:
Done